Libplanet.Net/Swarm.cs

```diff
@@ -208,10 +234,14 @@ public partial class Swarm<T> : IDisposable
 
         internal RoutingTable RoutingTable { get; }
 
+        internal RoutingTable MessageRoutingTable { get; }
+
         internal IProtocol PeerDiscovery { get; }
 
         internal ITransport Transport { get; private set; }
 
+        internal ITransport MessageTransport { get; private set; }
```

Comment:
How about using the name `ConsensusRoutingTable` and `ConsensusTransport`? It may be confusing for the word `Message`.